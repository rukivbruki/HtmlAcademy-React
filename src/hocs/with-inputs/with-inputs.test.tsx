import * as React from "react";
import * as renderer from "react-test-renderer";
import withInputs from './with-inputs';

interface MockComponentProps {
  children: React.ReactNode;
}

const MockComponent = (props: MockComponentProps) => {
  const {children} = props;

  return (
    <div>
      {children}
    </div>
  );
};

const MockComponentWrapped = withInputs(MockComponent);

it(`withInputs renders correctly`, () => {
  const tree = renderer.create((
    <MockComponentWrapped/>
  ), {
    createNodeMock() {
      return {};
    }
  }).toJSON();

  expect(tree).toMatchSnapshot();
});
