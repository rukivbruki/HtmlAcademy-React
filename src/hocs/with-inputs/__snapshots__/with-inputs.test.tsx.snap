// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`withInputs renders correctly 1`] = `<div />`;
